02-express-tutorial/app.js

```diff
@@ -1 +1,69 @@
-console.log('Express Tutorial')
+const express = require('express');
+const { products } = require('./data')
+
+const app = express();
+
+app.use(express.static('./public')) 
+
+app.get('/api/v1/test', (req,res)=> {
+  app.get(res.json({message: 'It worked!'}))
+})
+
+app.get('/api/v1/products', (req,res)=> {
+  const newProducts = products.map((product) => {
+    const { id, name, image } = product
+    return { id, name, image }
+  })
+  res.json(newProducts)
+})
+
+app.get('/api/v1/productsall', (req,res)=> {
+  app.get(res.json(products))
+})
+
+app.get('/api/v1/products/:productID', (req,res)=> {
+  const idToFind = parseInt(req.params.productID) // because this will be a string, and we need an integer
```

Comment:
You don't need comments to explain that. Since it is only for yourself, and you are learning, there is no problem whatsoever. But, a comment is something that makes the code more "crowded" and harder to read. You only need comments when it is something really complex that would take time for someone to understand.
Also, you can call the variable "id", since it is an endpoint for products, it is obviously a productId, you would probably only need to specify if it was an ID for another field. e.g.: You want to find the product by the user that created the product's ID.